### ISSUE TITLE
Type of Service (DSCP and ECN)

### ISSUE THREAD

**Issue by stephanbuys** [libpnet/libpnet#243]
Hi, there are some handy conversion charts for the IPv4 DSCP and ECN headers, would something like it be welcome in libpnet_macros? I'm thinking that the values could be expressed as an Enum or implemented as something like `header.get_dscp_expanded()` ?

Please see http://bogpeople.com/networking/dscp.shtml

**Comment by mrmonday** [comment]
Hi, thanks for your interest!

Adding proper enumerations for DSCP and ECN is definitely something we would like in libpnet - I'm willing to help mentor if this is something you're interested in adding. I would imagine it being implemented in a similar way to the `IpNextHeaderProtocol` structure.

**Comment by stephanbuys** [comment]
@mrmonday yeah, would be happy to try - a little swamped at the moment, but will ping back as soon as I have bandwidth again.
